code-examples/protect-page-login/expressjs/index.js

```diff
@@ -0,0 +1,22 @@
+const app = require("express")()
+// highlight-next-line
+const sdk = require("@ory/client-fetch")
+
+// highlight-start
+const ory = new sdk.FrontendApi(
+  new sdk.Configuration({
+    basePath: process.env.ORY_ENDPOINT || "http://localhost:4000/.ory",
+  }),
+)
+ // highlight-end
+
+app.get("/", (req, res) =>
+  // highlight-start
+  ory
+    .toSession({ cookie: req.header("cookie") })
+    .then((data) => res.json(data))
+    .catch(() => res.redirect("/.ory/ui/login")),
+  // highlight-end
```

Comment:
This can be as is, but realistically users will use Ory like a middleware so like:

```
app.get("/", requireAuth((req, res) => {
  // req.session.id
})
```

For the sake of getting this over the line and since it was like this before I think it's fine as is, but it probably isn't what people would need in a real world application.

Reply:
Got it. I think we should use middlewares since its the express standard. Will update.